<launch>
    <param name="/use_sim_time" value="false" />

    <arg name="udp_port" default="9763"/>
    <!-- The reference frame in the stream will have no translation or rotation, could be 'Pelvis' or 'T8' -->
    <arg name="ref_frame" default="T8"/>
    <!--  Scale the position of the pose if src_frame_id is not None.
          Its value equals to the robot/human body dimension ratio  -->
    <arg name="scaling" default="1"/>
    <!-- The publish rate is for publish the converted messages, it should not be larger than source rate -->
    <arg name="rate" default="60"/>

    <node pkg="roport" type="roport_xsens_server.py" name="roport_xsens_server" output="screen">
        <param name="udp_port" value="$(arg udp_port)"/>
        <param name="ref_frame" value="$(arg ref_frame)"/>
        <param name="scaling" value="$(arg scaling)"/>
        <param name="rate" value="$(arg rate)"/>
    </node>

</launch>